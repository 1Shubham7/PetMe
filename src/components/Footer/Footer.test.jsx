import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Footer from '.';

describe('Footer', () => {
  test('renders Footer', () => {
    const { getByText } = render(
      <BrowserRouter>
        <Footer />
      </BrowserRouter>,
    );
    expect(getByText(/Adopt/i)).toBeInTheDocument();
  });
});
