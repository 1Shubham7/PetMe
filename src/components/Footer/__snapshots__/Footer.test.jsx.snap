// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer > renders Footer with snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Footer
    </h1>
  </div>
</DocumentFragment>
`;
